client/src/components/Input/Input.js

```diff
@@ -10,9 +10,9 @@ const Input = styled.input`
   background: var(--basic-color);
   border: none;
   ::placeholder {
-    color: var(--emphasis-color);
+    color: rgba(0, 47, 214, 0.7);
```

Comment:
Why don't you use your color variables anymore?

Reply:
Because we're using opactity here, which only applies to the placeholders of the input fields